thunder/images/images.py

```diff
@@ -79,35 +80,44 @@ def toblocks(self, size='150', padding=None):
         from thunder.blocks.blocks import Blocks
         from thunder.blocks.local import LocalChunks
 
+        if chunk_size is 'auto':
+            chunk_size = str(int(1000000.0/self.shape[0]))
+
         if self.mode == 'spark':
-            chunks = self.values.chunk(size, padding=padding).keys_to_values((0,))
+            chunks = self.values.chunk(chunk_size, padding=padding).keys_to_values((0,))
 
         if self.mode == 'local':
             if isinstance(size, str):
                 raise ValueError("block size must be a tuple for local mode")
-            plan = (self.shape[0],) + tuple(size)
+            plan = (self.shape[0],) + tuple(chunk_size)
             chunks = LocalChunks.chunk(self.values, plan, padding=padding)
 
         return Blocks(chunks)
 
-    def toseries(self, size='150'):
+    def toseries(self, chunk_size='auto'):
         """
         Converts to series data.
 
         This method is equivalent to images.toblocks(size).toSeries().
 
         Parameters
         ----------
-        size : string memory size, optional, default = '150M'
-            String interpreted as memory size (e.g. '64M').
+        chunk_size : str or tuple, size of image chunk used during conversion, default = 'auto'
+            String interpreted as memory size (in kilobytes, e.g. '64').
+            The exception is the string 'auto', which will choose a chunk size to make the
+            resulting blocks ~100 MB in size. Tuple of ints interpreted as 'pixels per dimension'.
```

Comment:
Not obvious what a `block` is, maybe say `to make the resulting intermediate objects ~100MB in size`. Will still be confusing that a specification of `1000` is very different than what you get with `auto`, not sure what to do about that...